include/glow/Backend/BackendUtils.h

```diff
@@ -120,5 +120,15 @@ class RuntimeBundle {
 };
 } // namespace runtime
 
+bool isOutput(const Placeholder *PH);
+
+bool isInput(const Placeholder *PH);
+
+std::vector<std::tuple<Placeholder const *, bool, bool>>
```

Comment:
nit: typically it's const X * (and in other places)

Reply:
Add doxygen comment and explain what those methods do.

Reply:
> Add doxygen comment and explain what those methods do.

OK sure